packages/graphback-cli/package.json

```diff
@@ -54,6 +54,7 @@
   },
   "dependencies": {
     "@graphql-toolkit/code-file-loader": "0.9.7",
+    "graphql-serve": "0.11.0-rc5",
```

Comment:
Is this alright to do?

Reply:
Yeah it is because the module is private. You will need to make it public so that it can be published.